src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/GoogleDataprocDAO.scala

```diff
@@ -251,51 +252,51 @@ class GoogleDataprocDAO(protected val dataprocConfig: DataprocConfig, protected
   /**
     * Gets a dataproc Cluster from the API.
     */
-  private def getCluster(googleProject: GoogleProject, clusterName: String)(implicit executionContext: ExecutionContext): Future[GoogleCluster] = {
-    val request = dataproc.projects().regions().clusters().get(googleProject, dataprocConfig.dataprocDefaultRegion, clusterName)
-    executeGoogleRequestAsync(googleProject, clusterName, request)
+  private def getCluster(googleProject: GoogleProject, clusterName: ClusterName)(implicit executionContext: ExecutionContext): Future[GoogleCluster] = {
+    val request = dataproc.projects().regions().clusters().get(googleProject.string, dataprocConfig.dataprocDefaultRegion, clusterName.string)
+    executeGoogleRequestAsync(googleProject, "Cluster " + clusterName.string, request)
   }
 
   /**
     * Gets a compute Instance from the API.
     */
-  private def getInstance(googleProject: GoogleProject, zone: String, name: String)(implicit executionContext: ExecutionContext): Future[Instance] = {
-    val request = compute.instances().get(googleProject, zone, name)
-    executeGoogleRequestAsync(googleProject, name, request)
+  private def getInstance(googleProject: GoogleProject, zone: Zone, instanceName: InstanceName)(implicit executionContext: ExecutionContext): Future[Instance] = {
```

Comment:
this `Zone` is one of our `StringValueClass`es - could use Google's Zone instead?

Reply:
Do we ever have need of it outside Google-land? If not I'd say use Google's.

Reply:
As long as it's internal to GoogleDataprocDAO I'd say use Google types -- but we should use `StringValueClass`es in all interface methods.

Reply:
I looked into this a little more.  Our zone info comes from `google-api-services-dataproc`'s `getZoneUri()` as a URI `String`.  Google's `Zone` class doesn't use URIs and can't parse them, so I can't be sure it's an accurate representation of what we have here.

I'll keep a `StringValueClass` but name it `ZoneUri` to be more explicit about what it is.